assets/audio/sfx/special/piano_1.wav.import

```diff
@@ -0,0 +1,24 @@
+[remap]
+
+importer="wav"
+type="AudioStreamWAV"
+uid="uid://c45r1h0snq0cs"
+path="res://.godot/imported/piano_1.wav-a90def50795433467efe3a4f84781649.sample"
+
+[deps]
+
+source_file="res://assets/audio/sfx/special/piano_1.wav"
```

Comment:
Suggestion: cutting the audio file (first 200ish ms are silent so it feels "delayed" when played)